src/AppBundle/Search/Repository.php

```diff
@@ -53,14 +49,6 @@ public function getPullRequests($filters = []): array
         return $this->searchApi->issues($this->buildQuery($allFilters));
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function graphQL($query, $variables = []): array
-    {
-        return $this->graphQL->execute($query, $variables);
-    }
```

Comment:
Perhaps it's interesting to keep this part for future use?

Reply:
We can always re-revert this commit to have this feature back? ^^'